doc/manipulating_images/data_preparation.rst

```diff
@@ -10,30 +10,53 @@ Data preparation: loading and basic signal extraction
 
 |
 
-.. topic:: **File names as arguments**
+.. currentmodule:: nilearn.image
 
-   Nilearn functions and objects accept file names as arguments::
+File names or image objects
+=============================
+
+File names and objects for 3D and 4D images
```

Comment:
> > "time series of 3D images (i.e., 4D)"

> It makes a horribly long title though. Are you sure that it is that
> important?

I am detailing this wording below, in the text, rather than in the title.